ISL.ReIdentification.Core/Services/Foundations/Documents/IDocumentService.cs

```diff
@@ -20,5 +20,7 @@ public interface IDocumentService
         ValueTask AddContainerAsync (string container);
         ValueTask AddFolderAsync(string container, string folder);
         ValueTask<string> GetDownloadLinkAsync(string fileName, string container, DateTimeOffset expiresOn);
+        ValueTask<string> CreateDirectorySasTokenAsync(
```

Comment:
new line before multi-line piece of code
```suggestion

        ValueTask<string> CreateDirectorySasTokenAsync(
```

Reply:
Done

Reply:
fix